packages/registrar_client/src/config.ts

```diff
@@ -0,0 +1,18 @@
+export class Config {
```

Comment:
How is that used? The config params e.g the endpoint,mnemonic, .. etc should be passed to the client directly or through Config object that has that data, but populating that config is something completely different